File: 240416/일렬로 서있는 소 2/cattle-in-a-rowing-up-2.py
N = int(input())
cows = list(map(int, input().split()))

answer = 0
for i in range(len(cows)):
    for j in range(i + 1, len(cows)):
        for k in range(j + 1, len(cows)):
            if cows[i] <= cows[j] <= cows[k]:
                answer += 1

print(answer)

File: 260209/일렬로 서있는 소 2/cattle-in-a-rowing-up-2.cpp
#include <bits/stdc++.h>
using namespace std;


int main() 
{
    int N;
    cin >> N;
    vector<int> cows(N);
    for (int i = 0; i < N; i++) 
    {
        cin >> cows[i];
    }

    // Please write your code here.
    int answer = 0;
    for(int i = 0; i < cows.size(); i++)
    {
        for(int j = i + 1; j < cows.size(); j++)
        {
            for(int k = j + 1; k < cows.size(); k++)
            {
                if(cows[i] <= cows[j] && cows[j] <= cows[k])
                    answer += 1;
            }
        }
    }

    cout << answer;
    return 0;
}